charts/opentelemetry-demo-webstore/values.yaml

```diff
@@ -0,0 +1,116 @@
+observability:
+  otelcol:
+    enabled: true
+  jaeger:
+    enabled: true
+
+image:
+  repository: otel/demo
+  pullSecrets: []
+
+serviceAccount: ""
+
+components:
+  redis:
+    enabled: true
+    image: redis:alpine
+    ports:
+    - name: redis
+      value: 6379
+
+  ad-service:
```

Comment:
Can we make component names consistent? `-` is not the best symbol for values.yaml parameters. camelCase is usually recommended in helm ecosystem. But I think it's better to keep component names consistent with https://github.com/open-telemetry/opentelemetry-demo/tree/main/src and https://github.com/open-telemetry/opentelemetry-demo/blob/main/docker-compose.yml. So just removing dashes would be probably good.

Reply:
Because I want to automatically set environment variables like PRODUCT_CATALOG_SERVICE_ADDR by name.
If it's all lowercase, it won't be able to distinguish between words.
**I will change it to camel.** 
I'll move to the demo repo to discuss this later. (In the future, it is likely that value.yaml will be used to automatically generate docker-compose, thus avoiding similar problems)

Reply:
fixed it